### PR TITLE
Welcome screen layout

### PR DESCRIPTION
**Pull request by keisenbrand**
NOTE: this PR includes commits made by susannah for some reason -- shouldn't be an issue since the file additions are the same as what is currently in master. I pulled and rebased my branch, but something weird happened where i had to resolve merge conflicts and the commits got added to my branch.

UI for welcome screen:
IMG_7295

### CODE REVIEW

File: .expo/packager-info.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (lfvarela) [inline .expo/packager-info.json]: Try not to upload your .expo folder

File: volunta/components/CommunityProfileEventCardHorizontalScroll.js~HEAD [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (lfvarela) [inline volunta/components/CommunityProfileEventCardHorizontalScroll.js~HEAD]: Is this file from Sus?

File: volunta/screens/WelcomeScreen.js [hunk 3]
```diff
@@ -2,33 +2,66 @@ import React from 'react';
...
   render() {
     return (
       <View style={styles.container}>
-        <View style={styles.welcomeContainer}>
-          <Image
-            source={require('../assets/images/logo.png')}
-            style={styles.logo}
-          />
+        <Image
+          source={require('../assets/images/logo.png')}
```
Comment (lfvarela) [inline volunta/screens/WelcomeScreen.js]: Should we start adding constants at the top of files to get assets? Maybe some constant file for asset filepaths? Up to you but I'd like to start a pattern.

File: volunta/components/CommunityProfileEventCardHorizontalScroll.js~HEAD (hunk 2 above)
Comment (keisenbrand) [reply volunta/components/CommunityProfileEventCardHorizontalScroll.js~HEAD]: yes, explained in my first comment

File: volunta/screens/WelcomeScreen.js (hunk 3 above)
Comment (keisenbrand) [reply volunta/screens/WelcomeScreen.js]: yes agree

File: volunta/components/CommunityProfileEventCardHorizontalScroll.js~HEAD (hunk 2 above)
Comment (lfvarela) [reply volunta/components/CommunityProfileEventCardHorizontalScroll.js~HEAD]: Sorry missed that part

File: .expo/packager-info.json (hunk 1 above)
Comment (keisenbrand) [reply .expo/packager-info.json]: oops

Comment (keisenbrand) [reply .expo/packager-info.json]: not sure why it was included because of the gitignore

**Pull request merged**